src/components/connection_handler/src/connection.cc

```diff
@@ -294,6 +313,23 @@ void Connection::SetProtectionFlag(
     service_rpc->is_protected_ = true;
   }
 }
+
+bool Connection::IsSessionServiceExists(
```

Comment:
@AKalinich-Luxoft `Is...exists` - this is wrong. `Does exist` - this one is correct. Or `SessionServiceExists` at least

Reply:
@AByzhynar renamed in https://github.com/smartdevicelink/sdl_core/pull/1971/commits/a87c65251e81cb914deaa1f0fe1716175fe1270d